hybris/custom/klarnapaymentaddon/acceleratoraddon/web/src/com/klarnapayment/controllers/pages/checkout/KPOrderConfirmationController.java

```diff
@@ -66,12 +69,20 @@ public class KPOrderConfirmationController
 	@Resource(name = "kpCustomerFacade")
 	private KPCustomerFacade kpCustomerFacade;
 
+	@Resource(name = "cartService")
+	private CartService cartService;
+
+	@Resource(name = "cartRestorationStrategy")
+	private CartRestorationStrategy cartRestorationStrategy;
+
 	public static final String SESSIONID = "sessionId";
 	public static final String CLIENTTOKEN = "clientToken";
 
 	private static final String IS_KLARNA_EXP_CHECKOUT_SESSION = "isKlarnaExpCheckoutSession";
 	private static final String CLIENT_TOKEN = "clientToken";
 
+	Logger LOG = Logger.getLogger(KPOrderConfirmationController.class);
```

Comment:
Debug logging is enabled later on this file.

Reply:
Debug logs have been removed
Commit can be found in https://github.com/klarna/SAP-Commerce-klarna-payments/commit/c3d029e9830e600544e03277455cf385023f4a5c